src/finality/justification/decode.rs

```diff
@@ -241,17 +242,6 @@ impl<'a> From<PrecommitRef<'a>> for Precommit {
     }
 }
 
-impl fmt::Debug for Precommit {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("Precommit")
-            .field("target_hash", &self.target_hash)
-            .field("target_number", &self.target_number)
-            .field("signature", &&self.signature[..])
-            .field("authority_public_key", &self.authority_public_key)
-            .finish()
-    }
-}
```

Comment:
Context: this manual implementation existed because the `Debug` trait wasn't implemented on `[u8; 64]` before recently.